ui/pages/profile-view-stats.html

```diff
@@ -0,0 +1,189 @@
+                            <div class="article-card__content">
+                                <h3 class="article-card__content-title">
+                                    <a href="./articles-view.html">Automobile Industry Trends</a>
+                                </h3>
+                                <p class="article-card__content-text">
+                                    New cars to watch in 2019 are ultricies vitae risus ut aliquam. Quisque at maximus
+                                    justo, eu luctus lorem. Nullam mollis felis ligula, sit amet luctus sem luctus non
+                                    New cars to watch in 2019 are ultricies vitae risus ut aliquam. Quisque at maximus
+                                    justo, eu luctus lorem. Nullam mollis felis ligula, sit amet luctus sem luctus non
+                                </p>
+                                <div class="article-card__content-meta">
+                                    <span>5 min</span>
+                                    <span><i class="fa fa-eye"></i> 200</span>
+                                    <span><i class="fa fa-thumbs-up"></i> 15k</span>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="article-card profile">
+                            <div class="article-card__image"></div>
+                            <div class="article-card__content">
+                                <h3 class="article-card__content-title">
+                                    <a href="./articles-view.html">Automobile Industry Trends</a>
+                                </h3>
+                                <p class="article-card__content-text">
+                                    New cars to watch in 2019 are ultricies vitae risus ut aliquam. Quisque at maximus
+                                    justo, eu luctus lorem. Nullam mollis felis ligula, sit amet luctus sem luctus non
+                                    New cars to watch in 2019 are ultricies vitae risus ut aliquam. Quisque at maximus
+                                    justo, eu luctus lorem. Nullam mollis felis ligula, sit amet luctus sem luctus non
+                                </p>
+                                <div class="article-card__content-meta">
+                                    <span>5 min</span>
+                                    <span><i class="fa fa-eye"></i> 200</span>
+                                    <span><i class="fa fa-thumbs-up"></i> 15k</span>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-12 content-right">
+                            <a href="#" class="profile-meta__text">More Articles...</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        </div>
+    </section>
+</body>
+
+</html>
```

Comment:
@NkFab According to our code standard that we agreed on, an empty line is required at the and of every file.